build/build.ps1

```diff
@@ -244,12 +234,55 @@ function Stop-Processes() {
   Get-Process -Name "vbcscompiler" -ErrorAction SilentlyContinue | Stop-Process
 }
 
+function CreateBuildEnvScript()
+{
+  Create-Directory $ArtifactsDir
+  $scriptPath = Join-Path $ArtifactsDir "sdk-build-env.bat"
+  $scriptContents = @"
+@echo off
+title SDK Build ($RepoRoot)
+set DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
```

Comment:
I believe we used to have nix .sh files you could source. We should recover that too

Reply:
I think sourcing `build.sh - dogfood` should work, right?  So perhaps we don't need to generate separate shell scripts.